terraform/modules/galileo-eks/main.tf

```diff
@@ -139,30 +139,143 @@ module "eks_galileo" {
         AmazonEBSCSIDriverPolicy           = "arn:aws:iam::aws:policy/service-role/AmazonEBSCSIDriverPolicy",
       }
     }
-  }
+  }, 
+  var.create_ml_node_group ?
+    { 
+      galileo_ml = {
+        ami_type                   = "AL2_x86_64_GPU"
+        instance_types             = [var.ml_node_size]
+        name                       = "galileo-ml"
+        use_name_prefix            = false
+        use_custom_launch_template = false
+        create_launch_template     = false
+
+        disk_size = 200
+
+        min_size     = 1
+        max_size     = 5
+        desired_size = 1
+        labels = {
+          galileo-node-type = "galileo-ml"
+        }
+
+        tags = {
+          "k8s.io/cluster-autoscaler/${var.cluster_name}" = "owned",
+          "k8s.io/cluster-autoscaler/enabled"             = "true",
+        }
+
+        max_unavailable = 1
+
+        iam_role_additional_policies = {
+          AmazonEKSWorkerNodePolicy          = "arn:aws:iam::aws:policy/AmazonEKSWorkerNodePolicy",
+          AmazonEC2ContainerRegistryReadOnly = "arn:aws:iam::aws:policy/AmazonEC2ContainerRegistryReadOnly",
+          ClusterAutoscaler                  = "arn:aws:iam::${data.aws_caller_identity.current.account_id}:policy/ClusterAutoscaler_${var.cluster_name}",
+          AmazonS3FullAccess                 = "arn:aws:iam::aws:policy/AmazonS3FullAccess",
+          AmazonSSMManagedInstanceCore       = "arn:aws:iam::aws:policy/AmazonSSMManagedInstanceCore",
+          CloudWatchAgentServerPolicy        = "arn:aws:iam::aws:policy/CloudWatchAgentServerPolicy",
+          AmazonEBSCSIDriverPolicy           = "arn:aws:iam::aws:policy/service-role/AmazonEBSCSIDriverPolicy",
+        }
+      }
+    }
+  : {} 
+  )
 
   enable_irsa = true
+  enable_cluster_creator_admin_permissions = true
+}
+
+resource "random_password" "rds" {
+  count = var.create_rds_postgres_cluster ? 1 : 0
+  length  = 24
+  special = false
```

Comment:
Curious why this is false?

Reply:
![image](https://github.com/rungalileo/galileo-infra-blueprints/assets/50436466/914d427b-9017-4be7-be7b-62f6cff38777)


Some spl chars are not allowed. Randomly will hit the issue so I generally ignore special chars to avoid issue